Unity-Technologies/Graphics
# Standardization of SSR, SSGI and AO regarding naming (case 1335387)

### Purpose of this PR
Standardize naming of several volume components (case case 1335387):
- SSGI
- SS Reflection
- SS Refraction
- AO

---
### Testing status
Manual Test:
1. Checkout master and create an empty project
2. Added SSGI, SS Reflection, SS Refraction, AO volumes to a Volume profile
3. Added some custom values to them
4. Checkout this branch
5. > Observe that 
   a. We have all previous volumes with their associated data
   b. The naming of the volume components are correct.

Note: The SSGI volume component was renamed only after Unity was restarted.


Running Yamato tests as well.
* So far a few failure caused by `kTextureUploadUninitialized` that are also appearing on master.
* However 4082_DepthOfField-PB: Rendered failed, this one need investigation

---
### Comments to reviewers

Doc: I also moved the documentation pages as I changed the links as well.
